### PR TITLE
Docs: Update Acknowledgement & Citations

### PR DESCRIPTION
**Pull request by ax3l**
As discussed in today's developer meeting :)

Please feel free to add suggestions to the PR in case I missed papers.

### CODE REVIEW

**Review by mrowan137 — APPROVED** [review]

File: Docs/source/acknowledge_us.rst [hunk 1]
```diff
@@ -21,20 +21,52 @@ Please add the following sentence to your publications, it helps contributors ke
...
+  Primary WarpX contributors are with LBNL, LLNL, CEA-LIDYL, SLAC, DESY, CERN, and Modern Electron.
+  We acknowledge all WarpX contributors.
 
-Main WarpX reference
-********************
+Latest WarpX reference
+**********************
 
-.. raw:: html
+If your project even leads to a scientific publication, please consider citing the paper below.
```
Comment (dpgrote) [inline Docs/source/acknowledge_us.rst]: I would suggest "If your project leads to a scientific publication".
```suggestion
If your project leads to a scientific publication, please consider citing the paper below.
```

**Review by WeiqunZhang — APPROVED** [review]

File: Docs/source/acknowledge_us.rst [hunk 2]
```diff
@@ -21,20 +21,52 @@ Please add the following sentence to your publications, it helps contributors ke
...
+
+- Fedeli L, Zaim N, Sainte-Marie A, Thevenet M, Huebl A, Myers A, Vay J-L, Vincenti H.
+  **PICSAR-QED: a Monte Carlo module to simulate Strong-Field Quantum Electrodynamics in Particle-In-Cell codes for exascale architectures**. *New Journal of Physics*. in-press, 2022.
+  https://arxiv.org/abs/2110.00256
+
+- Zoni E, Lehe R, Shapoval O, Belkin D, Zaim N, Fedeli L, Vincenti H, Vay J-L.
+  **A Hybrid Nodal-Staggered Pseudo-Spectral Electromagnetic Particle-In-Cell Method with Finite-Order Centering**. under review, 2022.
+  https://arxiv.org/abs/2106.12919
+
+- Olga Shapoval O, Lehe R, Thevenet M, Zoni E, Zhao Y, Vay J-L.
```
Comment (EZoni) [inline Docs/source/acknowledge_us.rst]: I think the first name is repeated here:
```suggestion
- Shapoval O, Lehe R, Thevenet M, Zoni E, Zhao Y, Vay J-L.
```

**Review by MaxThevenet — APPROVED** [review]

**Comment by ax3l** [comment]
The new refs can also go here:
https://github.com/ECP-WarpX/WarpX/blob/development/.zenodo.json#L196-L198

Update: done

**Review by lucafedeli88 — APPROVED** [review]
Looks good to me! Thanks!

**Review by RemiLehe — APPROVED** [review]

**Pull request merged**